### PR TITLE
Add release with gh actions

### LINKED ISSUE #8: Automatically build and publish installer
Can we setup Github Actions to build any time the construct.yaml is chaned and then publish though something like https://github.com/features/packages? @xmnlab can you look at this with @scopatz guidance?

### PR DESCRIPTION
**Pull request by xmnlab**
Add a step to github actions that creates a release and uploads the package built by conda constructor

Resolves #8

### CODE REVIEW

**Comment by xmnlab** [comment]
ps: the CI is skipping the release step ... it will be triggered just when the tag is pushed.

File: README.md [hunk 1]
```diff
@@ -51,4 +51,16 @@ the `installer_type` key to have a value of `pkg` or `sh` depending on what you
 
 ## Gotchas
 
-The pkg installer seems to need to have `conda` itself listed as a spec right now.
\ No newline at end of file
+The pkg installer seems to need to have `conda` itself listed as a spec right now.
+
+## Releasing
```
[the remark this answers is not in the document]
Comment (xmnlab) [reply README.md:56]: sounds good. I will try that. thanks for the suggestion!

**Comment by xmnlab** [comment]
@scopatz I applied your recommendation about rever. it is ready again for review. thanks!

**Comment by xmnlab** [comment]
for the shell installer, I will work on that in a follow-up PR.

File: .github/workflows/package.yaml [hunk 2]
```diff
@@ -42,4 +43,27 @@ jobs:
...
\ No newline at end of file
+      run: constructor --conda-exe conda.exe constructor/
+
+    - name: get package name 
+      id: package_name
+      run: |
+        PACK_NAME=$(ls omnisci-*-MacOSX-x86_64.pkg)
+        echo "::set-output name=filename::$PACK_NAME"
+
+    - name: create release
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply .github/workflows/package.yaml:54]: `if: startsWith(github.ref, 'refs/tags/v')` limits when this is run. It will only run on release

**Review by kcpevey — APPROVED** [review]

**Pull request merged**